pages/index.js

```diff
@@ -1,7 +1,33 @@
+import useSWR from "swr";
+import Card from "@/components/Card";
+import styled from "styled-components";
+
 export default function HomePage() {
+  const { data } = useSWR("/api/plants");
+
+  if (!data) {
+    return <p>Loading...</p>;
+  }
+
```

Comment:
this is not a loading state, this is more an error. Please use the `isLoading` from useSWR to check for the loading state and display an error if Data is undefined.